rosa_architecture/rosa_architecture_sub/rosa-architecture-models.adoc

```diff
@@ -6,11 +6,11 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-ROSA architecture supports the following network configuration types:
+ROSA has two installation offerings with the architecture supporting the following network configuration types:
```

Comment:
This sentence confused me a little bit. It could be my lack of knowledge in this area. I think it is the "two installation offerings" part but then listing the configuration types. Maybe 
```suggestion
ROSA has two installation offerings. The architecture supports the following network configuration types:
```
If you could it might be good to list the installation offerings with that first sentence. Just a thought.